CHANGELOG.md

```diff
@@ -59,7 +60,7 @@
    * FIXED: Better hash function of GraphId [#3332](https://github.com/valhalla/valhalla/pull/3332)
 
 * **Enhancement**
-   * CHANGED: Favor turn channels more [#3222](https://github.com/valhalla/valhalla/pull/3222)
+   * CHANGED: Favor turn channels more [#3222](https://github.com/valhalla/valhalla/pull/3222)   
```

Comment:
i accidental white space but i cant stand to make ci build again

Reply:
sup sup, GH actions has a setting now where you can tell it to skip runs if it's only something trivial like docs changes, based on filepath pattern matching.

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-ignoring-paths

Reply:
It’s stupid circleci we use here mostly.. there’s been attempts to trick circle to skip stuff but IIRC wouldn’t work in all git scenarios. there’s been a branch (by Kevin I think) switching to actions. we do like PRs in that direction though, esp by semi-retired, experienced devs 😆(ain’t looking at no one in particular)

Reply:
@missinglink :wave:  yep we did actually add a couple hacks to our CI flow to detect when the changes were trivial so i think im actually talking crap above.... but @nilsnolde is right we'd like to get over to actions if we can. i did start a branch but never took it to the end, also that was back when actions just came out and it was the wild west a little bit..

Reply:
I can have a look, can you please give me access to push dev branches to this repo?
It's much easier to test if I can push up my own dev branch and open a DRAFT PR if that's cool wit youz?

```bash
/code/valhalla/valhalla ghactions ❯ git push origin ghactions:missinglink/ghactions
ERROR: Permission to valhalla/valhalla.git denied to missinglink.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
if you really want to sure i can add you. i will warn you though, it is significantly convoluted!

Reply:
so is every C project, ooo... 🔥 haha, I'll give it a crack, it can't be as bad as compiling spatialite as a loadable extension can it?

looks like most of the work is done already in the Docker scripts {famous last words}

Reply:
haha lovely! thanks a ton for even attempting! obviously let us know if there's questions around the convolutions